### PR TITLE
The message "No resources found" means success in this context.

### PR DESCRIPTION
**Pull request by guettli**
add one line to make installing more obvious.

### CODE REVIEW

File: content/docs/prologue/quick-start.md [hunk 1]
```diff
@@ -44,7 +44,10 @@ Once you have the files on your machine change into the project's root directory
 ```shell
 # Create the namespace and CRDs, and then wait for them to be availble before creating the remaining resources
 kubectl create -f manifests/setup
+
+# Wait until CRD "servicemonitors" was created. The message "No resources found" means success in this context.
```
Comment (simonpasquier) [inline content/docs/prologue/quick-start.md:48]:
```suggestion
# Wait until CRD "servicemonitors" was created. The message "No resources found" means success in this context.
```
```suggestion
# Wait until the "servicemonitors" CRD is created. The message "No resources found" means success in this context.
```

**Review by simonpasquier — COMMENTED** [review]
could you rebase? it should make the CI happy.

**Comment by guettli** [comment]
> could you rebase? it should make the CI happy.

I merged and committed your improvement. CI looks good now. Let me know if anything is missing.

**Review by simonpasquier — APPROVED** [review]
thanks!

**Pull request merged**